src/components/features/Brand/BrandBox.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+
```

Comment:
W ramch zacowania spójności projektowej nazwałbym ten plik BrandContainer, tak jak robimy to w wypadku innych sekcji tego typu

Reply:
pozostanę przy BrandBox